deploy/05-operator-subscription.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
yeah @prietyc123 this will need to be moved to a test specific deploy directory vs. this runtime on